internal/config/generator.go

```diff
@@ -190,6 +216,80 @@ func (c *JiraCLIConfigGenerator) configureInstallationType() error {
 	return nil
 }
 
+func (c *JiraCLIConfigGenerator) configureLocalAuthType() error {
+	var authType string
+
+	if c.usrCfg.AuthType == "" {
+		qs := &survey.Select{
+			Message: "Authentication type:",
+			Help:    "basic (login) or mtls (client certs)?",
+			Options: []string{"basic", "mtls"},
```

Comment:
Should here be `bearer` as well?

Reply:
I believe that bearer is only used in for InstallationType cloud